macros/edr/system/system_utils/full_names.sql

```diff
@@ -62,10 +62,13 @@
 
 
 {% macro relation_to_full_name(relation) %}
-    {%- if relation.database %}
+    {%- if relation.identifier.startswith('__dbt__cte__') %}
```

Comment:
Maybe use `relation.is_cte`?

Reply:
@haritamar 
Nice one, didn't see it before! 
I did leave the `is_ephemeral` macro to be more explicit, but changed it to use the `is_cte` flag